### DISCUSSION TITLE
Energy date picker start of week

### DISCUSSION THREAD

**Discussion by cybermaus** [home-assistant/frontend#12793] [Dashboard]
Is there a way to set the start of a week period in the [energy cards'](https://www.home-assistant.io/dashboards/energy/#energy-date-picker) `type: energy-date-selection`. 

I was digging through the source, and eventually did find some references to a variable that seemed to be related, but I cannot figure out where or how to set it.

Week start, depending on country, can be Sunday or Monday, but to be honest, I often want to easily pick part-week periods that are different then even that. (To be completely honest, I really would like a full date/time picker like on the history dashboard, but I could make due with a better starting day picker)

**Comment by emufan** [comment]
Agree. Would like to set the displayed timeframe as well. Esp with the new upcoming Comparing energy data function. That means twice.

Show daterange a till b.

And with new function: Compare daterange a till b to x till z (of course day-delta has to be same).

**Reply by cybermaus** [reply]
Note this was a question, not an enhancement request.
Maybe I do need to make it into one.

In the mean time, would you mind voting for a related enhancement? https://community.home-assistant.io/t/link-date-picker-and-stat-graph/421378/2
